packs/data/feats.db/command-corpse.json

```diff
@@ -0,0 +1,48 @@
+{
+    "_id": "vCsvT7xqBIolF7zH",
+    "data": {
+        "actionCategory": {
+            "value": ""
+        },
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "description": {
+            "value": "<p>You gain the @Compendium[pf2e.feats-srd.Command Undead]{Command Undead} feat, but you can use it only to transform the effects of <em>@Compendium[pf2e.spells-srd.Harm]{Harm}</em> spells you cast on mindless undead. These spells can control creatures with a level equal to or lower than your level – 1 (instead of your level – 3).</p>"
```

Comment:
```suggestion
            "value": "<p>You gain the @Compendium[pf2e.feats-srd.Command Undead]{Command Undead} feat, but you can use it only to transform the effects of <em>@Compendium[pf2e.spells-srd.Harm]{Harm}</em> spells you cast on mindless undead. These spells can control creatures with a level equal to or lower than your level -1 (instead of your level - 3).</p>"
```